config/operator/rbac/role.yaml

```diff
@@ -0,0 +1,84 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cl-operator-manager-role
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - endpoints
+  - namespaces
```

Comment:
seems to be overly permissive?
For example, why do we need to create namespaces, endpoints, etc. at runtime?

Reply:
I understand this is autogenerated so should be fixed at the source, where the annotations are defined.